src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
I am just a bit confused about the state of this PR. Is it accepted, rejected, or deferred until concerns about functionality/performance are addressed? I see neither of these, but just some ideas for further improvements (which are interesting, but the evaluation of the PR should not depend on those results unless they reveal a bug or the like).